pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -545,6 +548,10 @@ type HTTPConfig struct {
 	// Optional proxy URL.
 	// +optional
 	ProxyURL string `json:"proxyURL,omitempty"`
+	// FollowRedirects specifies whether the client should follow HTTP 3xx redirects.
+	// +optional
+	// +kubebuilder:default:=true
+	FollowRedirects bool `json:"followRedirects"`
```

Comment:
don't we want a follow-up PR to address the lack of `FollowRedirects` for all existing CRDs?

Reply:
yes, I think we can add it to `Prometheus` also. Again, I can add it here but it might be better to tackle the different API and code generation in a follow up?

Reply:
I agree, lets remove it and tackle `FollowRedirects` for all CRDs in a follow-up PR.

Reply:
I was thinking initially that since this is the only one that touches `v1alpha1` I would include it here and update and generate the `v1` API(s) together?

Reply:
my point is more to say that if we expose the field in the CRD then we need to wire it into `configGenerator.convertHTTPConfig()` + proper version sanitization.